misk-redis/src/main/kotlin/misk/redis/RedisClusterModule.kt

```diff
@@ -60,18 +61,22 @@ class RedisClusterModule @JvmOverloads constructor(
 
   @Provides @Singleton
   internal fun provideRedisClusterClient(
+    clientMetrics: RedisClientMetrics,
+    unifiedJedis: UnifiedJedis
+  ): Redis = RealRedis(unifiedJedis, clientMetrics)
+
+  @Provides @Singleton
+  internal fun provideUnifiedJedis(
     config: RedisClusterConfig,
     deployment: Deployment,
     metrics: Metrics,
     ticker: Ticker,
-  ): Redis {
+  ): UnifiedJedis {
     // Get the first replication group, we only support 1 replication group per service.
     val replicationGroup = config[config.keys.first()]
       ?: throw RuntimeException("At least 1 replication group must be specified")
 
     // Create our jedis pool with client-side metrics.
```

Comment:
I don't think `metrics` is wired up in here anymore?

Reply:
Thanks, removed that and `ticker`